docs/acl_guide.rst

```diff
@@ -0,0 +1,82 @@
+ACL
```

Comment:
Thanks for fixing the structure and moving the guides up. With this update, I plan to modify this guide to include both ACL and IAM. I'll create a separate PR once this is merged in :)